internal/payload_endpoints/main.go

```
@@ -1,31 +1,68 @@
+				})
+			} else {
+				log.Info("Payload handled successfully")
+				c.JSON(200, gin.H{
+					"response": res,
+				})
 			}
 		})
 	}
 
-	router.Run(":" + strconv.Itoa(port))
+	router.Run(":" + strconv.Itoa(cfg.PayloadEndpoints.Port))
```

golangci-lint (1.62.2)

Error return value of `router.Run` is not checked

(errcheck)